client/js/main.ts

```diff
@@ -354,6 +376,29 @@ document.getElementById("add-attendee")!.addEventListener("click", e => {
 		"name": nameInput.value.trim(),
 		"email": emailInput.value.replace(/, /g, ",").trim()
 	}).then(() => {
+		// Get current list of tags
+		let tags: string[] = Array.prototype.slice.call(document.querySelectorAll("#tag-choose > option")).map((el: HTMLOptionElement) => {
+			return el.textContent;
+		});
+		// Add new tags to the options list
+		let newTags: string[] = tagInput.value.trim().split(",").map((t) => { return t.trim().toLowerCase(); });
+		for (let i = 0; i < newTags.length; i++) {
```

Comment:
Ditto Pt. II

Reply:
Actually since this is repeated code, code this be put into a function instead of being duplicated?